src/modules/fw_att_control/ecl_yaw_controller.cpp

```diff
@@ -138,29 +143,31 @@ float ECL_YawController::control_bodyrate(const struct ECL_ControlData &ctl_data
 		lock_integrator = true;
 	}
 
-	/* input conditioning */
-	float airspeed = ctl_data.airspeed;
+	/* Close the acceleration loop if _coordinated_method wants this: change body_rate setpoint */
+	if (_coordinated_method == COORD_METHOD_CLOSEACC) {
```

Comment:
You could remove `_coordinated_method` entirely.

Reply:
Done